community/portworx/templates/hooks/pre-upgrade/retain-daemonset-install.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
   annotations:
     "helm.sh/hook": pre-upgrade
     "helm.sh/hook-weight": "10"
-    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+    "helm.sh/hook-delete-policy": before-hook-creation
```

Comment:
Why not remove the hook on successful upgrade?

Reply:
i removed it for debugging, added back